pgt/src/PGT/Output/Test/Comments.hs

```diff
@@ -34,14 +35,16 @@ instance Render MetaComment where
     ErrorMetaComment             -> "(ERROR)"
     RowCountMetaComment rowCount -> render rowCount
 
-data Comments = Comments
+data Commentary a = Commentary
   { metaComment :: MetaComment
-  , text        :: NonEmpty Text
+  , text        :: a
   }
   deriving stock (Eq, Show)
 
+type Comments = Commentary (NonEmpty Text)
```

Comment:
Comment: An incoming commit adds a new single-line comment i.e `type Comments = Commentary Text` and that's the rationale for the `Commentary` type.